package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typegoose",
-  "version": "5.5.1",
+  "version": "5.5.2",
```

Comment:
Given the fact that new string validation options have been merged, too, we have to release version 5.6.0.